src/AutoRest.CSharp/LowLevel/Output/LowLevelRestClient.cs

```diff
@@ -66,7 +67,7 @@ private IEnumerable<RestClientMethod> BuildAllMethods()
                     }
 
                     Request request = new Request (method.Request.HttpMethod, method.Request.PathSegments, method.Request.Query, method.Request.Headers, body);
-                    yield return new RestClientMethod (method.Name, method.Description, method.ReturnType, request, parameters.ToArray(), method.Responses, method.HeaderModel, method.BufferResponse, method.Accessibility);
+                    yield return new RestClientMethod (method.Name, method.Description, method.ReturnType, request, parameters.ToArray(), method.Responses, method.HeaderModel, method.BufferResponse, accessibility);
```

Comment:
pull it off `method.BufferResponse`?

Reply:
Not sure what you are asking here?

Reply:
Sorry. You seem to be passing accessibility both into BuildMethod and here. Why both?

Reply:
We're calculating the "base" method via _builder.BuildMethod and then clone it with modifications.

I think the way I'm doing it is technically correct, but just copying it over with:

```yield return new RestClientMethod (method.Name, method.Description, method.ReturnType, request, parameters.ToArray(), method.Responses, method.HeaderModel, method.BufferResponse, method.Accessibility);
```

would be more clear.